distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+
+[IMPORTANT]
+====
+Although ${branding} can be installed by any user, it is recommended for security reasons to have a non-`root` user execute the ${branding} installation.
+====
+
+=== ${branding} Installation Prerequisites
+
+These are the system/environment requirements to configure _prior_ to an installation.
+
+=== Installing With the ${branding} Distribution Zip
+
+To install the ${branding} distribution zip, perform the following:
+
+* After the prerequisites have been met, as a root user if for *NIX, change the current directory to the desired install location.
+This will be referred to as `<INSTALL_DIRECTORY>`.
```

Comment:
I landed on going with <INSTALL_DIRECTORY> globally.

and changed to ${branding-lowercase}